### ISSUE TITLE
MGS3 HD Cutscene Model missing? (Minor)

### ISSUE THREAD

**Issue by wilbarker5** [RPCS3/rpcs3#10657]
In a cutscene where Snake and Eva attempt to escape, the jet they steal doesn't load properly in a certain scene, it's extremely minor but thought I'd share.

Log for MGS HD Collection
[RPCS3.log](https://github.com/RPCS3/rpcs3/files/6935240/RPCS3.log)

![missing model](https://user-images.githubusercontent.com/49188770/128275567-adff20a3-74a1-4fb2-97b3-46f960ecc072.png)

**Comment by WilFitzz** [comment]
bump

**Comment by Jonathan44062** [comment]
RSX Capture, the issue is still there
https://www.mediafire.com/file/tafrgyzlabzvcly/BLUS30847_20220324224928_capture.rar/file

**Comment by Asinin3** [comment]
Does this still happen with zcull set to precise and strict rendering mode?

**Comment by Jonathan44062** [comment]
![image](https://user-images.githubusercontent.com/78843189/164568361-620bca6c-5593-4b95-a94f-8700342f28a8.png)
Never changed in ZCULL Acurracy, so, I need to see the strict rendering mode

**Comment by Jonathan44062** [comment]
![image](https://user-images.githubusercontent.com/78843189/164571816-6839d402-9b19-4c26-a140-e68f9e3cb206.png)
Nope, using Strict Rendering Mode is still there
Log file
[RPCS3.log.gz](https://github.com/RPCS3/rpcs3/files/8535868/RPCS3.log.gz)

**Comment by Jonathan44062** [comment]
Yes
![image](https://user-images.githubusercontent.com/78843189/169371513-93e0b881-7fa0-4820-b145-934e3ecce65d.png)
Also, what build is that? Due to there's no number on it, using a custom build is not allowed

**Comment by Xcedf** [comment]
@Jonathan44062 well, here's normal build if you say so
perhaps some of your settings not right
![30](https://user-images.githubusercontent.com/7060586/169373405-e63a9aba-f8e3-4319-9d94-73d3c1083e3f.jpg)

**Comment by Jonathan44062** [comment]
My settings is default, and Enabing Read color buffers, other than that, no
Maybe is fixed for the PSN version instead of the Disc version?, in this case, MGS Collection

**Comment by WilFitzz** [comment]
> @Jonathan44062 well, here's normal build if you say so perhaps some of your settings not right ![30](https://user-images.githubusercontent.com/7060586/169373405-e63a9aba-f8e3-4319-9d94-73d3c1083e3f.jpg)

Yeah it must be the HD collection disc version thats buggered compared to the HD PSN version you are using.

**Comment by RainbowCookie32** [comment]
Doubt there's a difference between PSN and Disc versions. Different GPUs maybe?

**Comment by WilFitzz** [comment]
> Doubt there's a difference between PSN and Disc versions. Different GPUs maybe?

Got an rtx 3070 founders edition, the log should correspond to what driver i was using.

**Comment by WilFitzz** [comment]
What is the status of this issue? Has this issue been resolved coincided with any others?

**Comment by Jonathan44062** [comment]
First try
![image](https://user-images.githubusercontent.com/78843189/215200325-949f9912-c7d3-4fa8-a2c8-d9598fd0e2b2.png)
Third try
![image](https://user-images.githubusercontent.com/78843189/215204877-4692693e-5a75-447f-ae37-fcc783df31b7.png)
Still an issue, but it seems to be random, not a 100% fix.

**Comment by digant73** [comment]
@Jonathan44062 is that still an issue?

**Comment by Jonathan44062** [comment]
<img width="2559" height="1391" alt="Image" src="https://github.com/user-attachments/assets/e83f9d22-1e74-4d05-bb46-8870afb857d0" />
Still an issue

RSX Capture: https://www.mediafire.com/file/vans3xmabkbt97t/BLUS30847_20251103153209_capture.rrc.gz/file

**Comment by ghost** [comment]
Is this issue still present?

**Comment by digant73** [comment]
please an update on current build

**Comment by Jonathan44062** [comment]
<img width="2559" height="1389" alt="Image" src="https://github.com/user-attachments/assets/c82c7c5d-2949-4605-bd10-4092f9618155" />
Still broken, just checked in theater demo
